New package: Datasets v0.2.5

JuliaRegistrator opened the pull request:
- Registering package: Datasets
- Repository: https://github.com/awi-esc/Datasets.jl
- Created by: @perrette
- Version: v0.2.5
- Commit: 841dc4fcf3d74aeec41120ab1459dc31a29efa93
- Git reference: HEAD
- Description: Keep track of datasets used in a project

goerz commented:
See https://github.com/JuliaRegistries/General/pull/132410#issuecomment-2947961222 about the package name.

JuliaHubRegistrator committed:
New package: Datasets v0.2.5

UUID: b8ee69ef-a20a-4d38-bceb-a68d72817f72
Repo: https://github.com/awi-esc/Datasets.git
Tree: f7d4da65acae87e9b6d4ead57cbff9bbbdc4debf

Registrator tree SHA: d5716b7a540e5fbc43640c2fff2906fe50e9525a
